app/package.json

```diff
@@ -20,8 +20,9 @@
     "test": "npm run test:unit && npm run test:e2e",
     "test:coverage": "live-server ./coverage/lcov-report",
     "test:coverage-branch": "npm run test:unit -- --coverage && git diff origin/develop | diff-test-coverage -c ./coverage/lcov.info -t lcov --",
-    "test:e2e:browserstack": "nightwatch -c tests/e2e/browserstack.conf.js -e windows",
+    "test:e2e:browserstack": "npx env-cmd nightwatch -c tests/e2e/browserstack.conf.js -e windows --fe='https://app.lunie.io' --api=https://staging-api.lunie.io",
```

Comment:
this would test the production version not the current version

Reply:
how to make it dynamic? Is there a way?

Reply:
instead of doing this script, just document in the README if it is not already

Reply:
OK, I will add it